examples/validation-app/src/__tests__/acceptance/validate.acceptance.ts

```diff
@@ -62,14 +101,14 @@ describe('validate properties', () => {
     );
   });
 
-  it('fails when creating a CoffeeShop with an invalid type', async () => {
+  it('should use sequence-level custom error handler', async () => {
     const invalidCapacityTypeCS = {
       city: 'Toronto',
       phoneNum: '416-111-1111',
       capacity: '10',
     };
     const response = await client
-      .post('/coffee-shops')
+      .patch('/coffee-shops')
```

Comment:
just wondering why we're changing this to `patch`? the intention was to create a CoffeeShop.

Reply:
https://github.com/strongloop/loopback-next/pull/5937#discussion_r456410548